### PR TITLE
RISC-V: Replace mentions of BBL with OpenSBI

### PR DESCRIPTION
**Pull request by alistair23**
This is part of a collection of PRs to convert from using the legacy BBL and riscv-pk to OpenSBI.

Other PRs include:
* https://github.com/seL4/seL4/pull/272
* https://github.com/seL4/sel4test/pull/28
* https://github.com/seL4/seL4_tools/pull/46
* https://github.com/seL4/sel4test-manifest/pull/7

Signed-off-by: Alistair Francis <alistair.francis@wdc.com>

### CODE REVIEW

File: include/plat/spike/plat/instance/rocket-chip/hardware.h [hunk 1]
```diff
@@ -30,7 +30,7 @@ static inline void plic_init_controller(void)
 /* Available physical memory regions on platform (RAM minus kernel image). */
 /* NOTE: Regions are not allowed to be adjacent! */
 static p_region_t BOOT_DATA avail_p_regs[] = {
-    /* The first 2MB are reserved for the SBI in the BBL */
+    /* The first 2MB are reserved for OpenSBI */
```
Comment (yyshen) [inline include/plat/spike/plat/instance/rocket-chip/hardware.h:33]: @alistair23 what is the memory region reserved by OpenSBI?

**Comment by yyshen** [comment]
@ssrg-bamboo `test`

**Comment by ssrg-bamboo** [comment]
Hello, I'm a bot! I'll bring this PR into Trustworthy Systems and run some tests

**Comment by ssrg-bamboo** [comment]
All the tests we ran have passed! Nice job!

File: include/plat/spike/plat/instance/rocket-chip/hardware.h (hunk 1 above)
Comment (alistair23) [reply include/plat/spike/plat/instance/rocket-chip/hardware.h:33]: It should also be only 2MB.

**Review by szhuang — APPROVED** [review]

**Review by axel-h — APPROVED** [review]

**Comment by alistair23** [comment]
The failing tests seem to be unrelated to this PR.

**Review by axel-h — APPROVED** [review]

**Comment by axel-h** [comment]
@ssrg-bamboo test

**Comment by axel-h** [comment]
> The failing tests seem to be unrelated to this PR.

Yes, I'm seeing similar failures on my PRs also.

**Comment by alistair23** [comment]
Ping!

**Comment by alistair23** [comment]
Ping!

**Comment by axel-h** [comment]
I got told this is in the queue for internal testing.

File: include/plat/spike/plat/instance/rocket-chip/hardware.h (hunk 1 above)
Comment (axel-h) [inline include/plat/spike/plat/instance/rocket-chip/hardware.h:33]: I wonder, is this files still used anywhere?

**Comment by ssrg-bamboo** [comment]
Hello, I'm a bot! I'll bring this PR into Trustworthy Systems and run some tests

**Comment by oliver-wm** [comment]
@ssrg-bamboo `test`

**Comment by ssrg-bamboo** [comment]
Hello, I'm a bot! I'll bring this PR into Trustworthy Systems and run some tests

**Comment by lsf37** [comment]
The test results on the GitHub checks are from before rebasing -- @oliver-wm has rebased and merged the PR internally and it should come out when sel4test and proofs have passed.

**Pull request closed without merging**